dexs/aerodrome-slipstream/index.ts

```diff
@@ -23,8 +24,43 @@ const abis = {
   fee: 'uint256:fee'
 }
 
+const PRE_LAUNCH_TOKEN_PRICING = {
+  // SYND token configuration
+  '0x11dc28d01984079b7efe7763b533e6ed9e3722b9': {
+    decimals: 18,
+    ticker: 'SYND',
+    conversionRate: 1.5887, // close VWAP price of SYND launch day on Sept 19, 2025
+    cutoffTimestamp: 1758240000, // September 19, 2025 00:00:00 UTC
+    description: 'SYND to USDC conversion at launch VWAP price'
+  }
+  // Future pre-launch tokens can be added here following the same structure:
+  // '0xtoken_address': {
+  //   decimals: decimals of the token,
+  //   conversionRate: price_per_token,
+  //   cutoffTimestamp: unix_timestamp,
+  //   description: 'Token description'
+  // }
+}
+
+const handlePreLaunchTokenConversion = (
+  token: string, 
+  amount: string, 
+  currentTimestamp: number, 
+  dailyBribesRevenue: sdk.Balances
+): boolean => {
```

Comment:
what I would do here is, instead of returning boolean, I would either add token value if config is missing or compute usd value and add it, this way, bribe data is handled in a single place, also, would move this to the same common file as the code is repeated.